app/metrics.yaml

```diff
@@ -5445,3 +5445,35 @@ set_default_setting_experiment:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2021-10-01"
+
+start_on_home:
```

Comment:
I think you need to move this all under the metrics ping.

Reply:
Actually scratch that from looking at https://github.com/mozilla-mobile/fenix/issues/19976. I am unsure what to do at the time of writing.